packages/react-property/src/html/index.js

```diff
@@ -0,0 +1,22 @@
+var attributeToProperty = require('../../lib/html/attribute-to-property');
+var attributes = require('../../lib/html/attributes');
+var booleanProperties = require('../../lib/html/boolean-properties');
+var overloadedBooleanProperties = require('../../lib/html/overloaded-boolean-properties');
+
+var attributeMap = {};
+var attributeName;
```

Comment:
Nice 👍 I haven't used `var` for such a long time I probably would have forgotten the scoping complications of `var` and loops.

Reply:
Thanks. I'm considering rewriting this to TypeScript but that can be an enhancement done later.